WPMedia/fusion-news-theme-blocks
# TMEDIA-362: Add Subheadline block to StoryBook

## Description
Add Subheadline block to StoryBook

## Jira Ticket
- [TMEDIA-362](https://arcpublishing.atlassian.net/browse/TMEDIA-362)

## Acceptance Criteria
Add the following blocks to the Blocks storybook, and set them up for visual regression testing with Chromatic: 
  * Subheadline
    
## Test Steps
1. Checkout this branch `git checkout TMEDIA-362-subheadline-storybook`
2. Run `npm run storybook` and check the Subheadline category in Blocks.

Since there were component changes:
1.  Run `npx fusion start -f -l @wpmedia/subheadline-block`
2. Validate render on http://localhost/pagebuilder/editor/curate?p=p0ciQAinp2VIRr&v=vFzdS81np2VIRr
  * check Subheadline block below:
  
    * ![image](https://user-images.githubusercontent.com/2287238/128251389-956a2384-abf7-49c9-9902-8130b39a7366.png)
  * Edit block and validate the render states
    * ![image](https://user-images.githubusercontent.com/2287238/128251502-16ef1625-33e6-4c7a-90f9-f533d8514240.png)

## Dependencies or Side Effects
N/A

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
